dashboard/db/migrate/20210311205255_remove_script_level_id_from_teacher_feedbacks.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveScriptLevelIdFromTeacherFeedbacks < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :teacher_feedbacks, :script_level_id
```

Comment:
There isn't a risk for production impact here is there? I don't think script levels is big enough but thought I'd check

Reply:
@sureshc had the same question on a previous PR: https://github.com/code-dot-org/code-dot-org/pull/38848 TL;DR: the table is large enough to be concerned, so we tested it out a previous migration on a production clone first. We have since shipped a couple migrations to this table without incident.

Reply:
to be clear we are talking about the size of teacher_feedbacks (2M), not script_levels.

Reply:
ah right 🤦‍♀️ sounds good thanks!